.gitlab-ci.yml

```diff
@@ -174,23 +174,26 @@ flatpak:
       - artifacts
     expire_in: 2 week
 
-#macos-dmg:
-#  rules:
-#    - if: '$CI_PIPELINE_SOURCE == "push"'
-#  tags:
-#    - mac
-#  stage: build
-#  script: 
-#    - echo Hello Mac
-#    - chmod +x ci-scripts/OSX/build.sh 
-#    - ci-scripts/OSX/build.sh /Users/work/Qt/5.9.9/clang_64/bin/ master "enable_log"
-#  after_script:
-#    - wget $icon_server/checkmark/$CI_COMMIT_REF_NAME/$CI_COMMIT_SHA/$CI_JOB_NAME/$CI_JOB_STATUS/${CI_JOB_URL#*/*/*/}
-#  artifacts:
-#    paths:
-#      - artifacts
-#    expire_in: 2 week
-#  allow_failure: true
+macos-dmg:
+  image: docker.io/sickcodes/docker-osx:latest
+  rules:
+    - if: '$CI_PIPELINE_SOURCE == "push"'
+  tags:
+    - docker
+  stage: build
+  script: 
+    - echo Hello Mac
+    - pwd
+    - sleep 3h
+    - chmod +x ci-scripts/OSX/build.sh 
+    - ci-scripts/OSX/build.sh /Users/work/Qt/5.9.9/clang_64/bin/ master "enable_log"
+  after_script:
+    - wget $icon_server/checkmark/$CI_COMMIT_REF_NAME/$CI_COMMIT_SHA/$CI_JOB_NAME/$CI_JOB_STATUS/${CI_JOB_URL#*/*/*/}
+  artifacts:
+    paths:
+      - artifacts
+    expire_in: 2 week
+  allow_failure: true
```

Comment:
This change with including `macos-dmg` section refers to something else, is it not?
Looking at the delays I understand this is some WIP code.

Reply:
yes, this change was made by accident - fixed